src/home/ReadAboutUs.jsx

```diff
@@ -0,0 +1,42 @@
+import React from "react";
+import "./ReadAboutUs.css";
+import card_wallpaper_1 from './../assets/images/card_wallaper_1.svg'
+import card_wallpaper_2 from './../assets/images/card_wallaper_2.svg'
+import card_wallpaper_3 from './../assets/images/card_wallaper_3.svg'
+import arrow from './../assets/images/long left.svg'
+export default function Home() {
+
+    return (
+        <section id="read-about-us">
+          <h1 className="heading"><span style={{color:'#ff3300'}}>Read</span> about us</h1>
+          <div className="container-aboutus">
+            <div className="content_aboutus">
+            <div className="card_aboutus">
```

Comment:
@naveen8801 Can you please make a seperate card component for the About Us cards so that we don't have the repetition of code.

Reply:
Ya sure , i'll :))